services/system/Accounts/plugin/accounts/wit/impl.wit

```diff
@@ -19,6 +19,8 @@ interface admin {
 
     /// Gets a list of all accounts that are known by the accounts app on this device.
     get-all-accounts: func() -> list<string>;
+
+    logout: func();
```

Comment:
There's already a `logout()` function in the `active-app` interface

Reply:
right. 2 different logins confused me. I removed this extra fn.